src/content/glossary/index.md

```diff
@@ -418,6 +510,10 @@ A [wallet](#wallet) using the hierarchical deterministic (HD) key creation and t
 
 A value used to generate the master [private key](#private-key) and master chain code for an HD [wallet](#wallet). The wallet seed can be represented by mnemonic words, making it easier for humans to copy, back up, and restore private keys.
 
+### hexadecimal {#hexadecimal}
+
+Common representation format for byte sequencing. Its advantage is that values are represented in a compact format using two characters per byte (the characters [0-9][a-f]).
```

Comment:
Not convinced we should add this.

Our listing criteria:
- [ ] This is a term not already found in the glossary (for similar terms, please consider the benefits of a new term vs updating an existing term)
- [ ] This term/definition is not a product advertisement or contain other promotional content
- [ ] **This term/definition is directly relevant to Ethereum**

We should be careful not to try to cram every computer science term into the glossary.

Reply:
ok! removed.